third-culture-software/bhima
# Shipment updates

This PR updates several things with shipments.
- Adds per-item unit weight to shipping items (value defaults to the inventory unit_weight but  can be overridden)
- Updates to the Shipments "overview" timeline (still needs a bit more work)
- Converts the shipment document (via the shipment registry action menu) to a report usable during actual shipments. 
   - Adds several fields to shipments (transport_mode, receiver)
   - Based on feedback from projects
   - Removes old delivery timeline from this document
   - Sample version attached: [Sample-BHIMA-shipment-document.pdf](https://github.com/IMA-WorldHealth/bhima/files/9112546/Sample-BHIMA-shipment-document.pdf)

Closes https://github.com/IMA-WorldHealth/bhima/issues/6732
Closes https://github.com/IMA-WorldHealth/bhima/issues/6734
Closes https://github.com/IMA-WorldHealth/bhima/issues/6721

**TESTING**
- Create a shipment
   - Add notes
   - Add info on the receiver and transport mode
   - Save the shipment
- Edit origin depot and destination depots (under Stock > Depot management) to add address/contact info that will appear in the shipment TO/FROM boxes.
- To get the depot address automatically added to the shipment TO/FROM box, edit a depot and "Join a location" to it
- In the Shipment Registry, use the action menu for a shipment and choose the "Document" option to see the report
